pyproject.toml

```diff
@@ -24,6 +23,12 @@ dependencies = [
     "urllib3>=1.26,<3",
     "pyecma376-2>=0.2.4"
 ]
+# This line indicates that the version is dynamically determined
```

Comment:
This comment doesn't say anything about how the dynamic versioning works, rather it states what is obvious from reading the code. 
 
It'd be better the explain here that we use `setuptools_scm` to dynamically determine the version based on the git tag, as you explain in the PR.